### DISCUSSION TITLE
Anki Report

### DISCUSSION THREAD

**Discussion by BCFCODE** [Sepehrsalimian/WebStarter#31] [General]
Send me a screenshot of your FCC Anki decks so I can see your progress and help you schedule your study program! 📸✨ @Sepehrsalimian #Task

**Comment by Sepehrsalimian** [comment]
![image](https://github.com/user-attachments/assets/4e6b2528-7136-448a-a949-bb895307721a)
![image](https://github.com/user-attachments/assets/b16a9888-2ff3-46de-93aa-8e3136d0378f)
![image](https://github.com/user-attachments/assets/78894a49-b813-4ef8-affe-17efe064ed12)
![image](https://github.com/user-attachments/assets/78b7a05d-a657-44aa-8f3d-d042a5163a57)
@BCFCODE
